service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/service/EncounterReportBundleService.java

```diff
@@ -108,7 +111,24 @@ private void addOrganization(Bundle bundle, Organization organization) {
 
     private static void addEntry(Bundle bundle, Resource resource) {
         bundle.addEntry()
-            .setFullUrl(resource.getIdElement().getValue())
-            .setResource(resource);
+                .setFullUrl(resource.getIdElement().getValue())
+                .setResource(resource);
+    }
+
+    private void addSubject(Bundle bundle, Encounter encounter) {
+        if (encounter.getSubjectTarget() instanceof Patient) {
+            Patient patient = (org.hl7.fhir.dstu3.model.Patient) encounter.getSubjectTarget();
+            addEntry(bundle, patient);
+        }
+        if (encounter.getSubjectTarget() instanceof Group) {
+            Group group = (Group) encounter.getSubjectTarget();
+            addEntry(bundle, group);
+            for (Group.GroupMemberComponent groupMemberComponent : group.getMember()) {
+//                addEntry(bundle, groupMemberComponent);
```

Comment:
why is this commented out?

Reply:
I was using the method to remove duplicated lines. This case does not match the others but line removed.